README.md

```diff
@@ -194,6 +194,35 @@ When using `make` to build the project, add this to the command line:
 
     CONAN_OPTIONS="-c tools.build:skip_test=1"
 
+## Building with Bazel
+
+Bazel tooling is currently experimental, and is meant to support users
+who already are using Bazel in their projects and are not afraid of
+providing their own registries and modules for Cloe's dependencies.
+
+(That allows us to not have to vendor massive amounts of Bazel modules
+in this repository, of which Boost is the main problematic point.
+Once Boost makes it into the Bazel Central Registry, it may be worthwhile
+to vendor the remaining libraries in this repository.)
+
+You will need to create a `.bazelrc.user` file in the repository root
+with the following contents:
+
+    common --registry=file:///path/to/your/bazel/registry
+
+This file is ignored by Git to prevent you from exposing your secrets.
+The registry should contain the following modules:
+
+    boost
+    cli11
+    incbin
+    inja
+    oatpp
+    open-simulation-interface
+    sol
```

Comment:
ah nevermind my ohter comment but maybe add the fact that apart from boost those other modules can be provided easily and link ecplicitly to the boost pr for the bcr

Reply:
Bazel support is experimental and requires the user to know what they are doing. If someone already has a Bazel infrastructure I trust they will find the dependencies.

The other modules can be provided easily is somewhat of an understatement if you are not already using Bazel. And if you are, then you likely already know. But maybe I put this in the readme.

Reply:
It's already in the README.

Reply:
(See my other comment.)

Reply:
Sure I am implying a somewhat familiarity with bazel here